### DISCUSSION TITLE
Can we add ReClass or YClass like features?

### DISCUSSION THREAD

**Discussion by Happy-Secret** [Flipboard/FLEX#658] [Ideas]
Can we add “Inspect and recreate in-memory data structures” like ReClass and YClass?

ReClass - https://github.com/ajkhoury/ReClassEx
YClass - https://github.com/ItsEthra/yclass

**Comment by NSExceptional** [comment]
Ham can you tell me more about your use case?

**Reply by Happy-Secret** [reply]
Sometime, we want to understand how data are being stored in memory..
With this memory layout understand in place, we can access specific data in an object using memory offset.
